x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -397,27 +397,29 @@ export const CaseComponent = React.memo<CaseProps>(
                       userCanCrud={userCanCrud}
                     />
                     {(caseData.type !== CaseType.collection || hasDataToPush) && (
```

Comment:
Just want to confirm that sub cases would be rendered the same UI as case individual right?

Reply:
Yes, that's right!